server/core/scheduler/service/src/main/java/com/antgroup/openspg/server/core/scheduler/service/common/SchedulerCommonService.java

```diff
@@ -0,0 +1,170 @@
+/*
+ * Copyright 2023 Ant Group CO., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied.
+ */
+package com.antgroup.openspg.server.core.scheduler.service.common;
+
+import com.antgroup.openspg.common.util.SchedulerUtils;
+import com.antgroup.openspg.server.common.model.exception.SchedulerException;
+import com.antgroup.openspg.server.common.model.scheduler.SchedulerEnum.InstanceStatus;
+import com.antgroup.openspg.server.common.model.scheduler.SchedulerEnum.TaskStatus;
+import com.antgroup.openspg.server.common.service.spring.SpringContextHolder;
+import com.antgroup.openspg.server.core.scheduler.model.service.SchedulerInstance;
+import com.antgroup.openspg.server.core.scheduler.model.service.SchedulerJob;
+import com.antgroup.openspg.server.core.scheduler.model.service.SchedulerTask;
+import com.antgroup.openspg.server.core.scheduler.model.task.TaskExecuteContext;
+import com.antgroup.openspg.server.core.scheduler.model.task.TaskExecuteDag;
+import com.antgroup.openspg.server.core.scheduler.service.metadata.SchedulerInstanceService;
+import com.antgroup.openspg.server.core.scheduler.service.metadata.SchedulerJobService;
+import com.antgroup.openspg.server.core.scheduler.service.metadata.SchedulerTaskService;
+import com.antgroup.openspg.server.core.scheduler.service.task.TaskExecute;
+import com.antgroup.openspg.server.core.scheduler.service.task.async.AsyncTaskExecute;
+import com.antgroup.openspg.server.core.scheduler.service.translate.TranslatorFactory;
+import com.google.common.collect.Lists;
+import java.util.Date;
+import java.util.List;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.collections4.CollectionUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.lang3.time.DateUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.util.Assert;
+
+/** Scheduler Common Service */
+@Service
+@Slf4j
+public class SchedulerCommonService {
+
+  public static final String UNDERLINE_SEPARATOR = "_";
+
+  @Autowired SchedulerJobService schedulerJobService;
+  @Autowired SchedulerInstanceService schedulerInstanceService;
+  @Autowired SchedulerTaskService schedulerTaskService;
+  @Autowired SchedulerValue schedulerValue;
+
+  /** set Instance To Finish */
+  public void setInstanceFinish(
+      SchedulerInstance instance, InstanceStatus instanceStatus, TaskStatus taskStatus) {
+    Long finish = 100L;
+    Date finishTime = (instance.getFinishTime() == null ? new Date() : instance.getFinishTime());
```

Comment:
What does the value of 100 represent? Please provide annotations for clarification.